leap_year.rb

```diff
@@ -0,0 +1,19 @@
+def leap_year(i)
+    puts "End Year"
+    o = gets.chomp
+    o = o.to_i
+    for a in i..o do
```

Comment:
You need to ask users for `start_year` and `end_year` and loop through it. Something like -
`(start_year..end_year).each do |year|`
       `if ..........             `          
            `......`
        `end`
`end`